strings/en/levels.json

```
@@ -4,5 +4,6 @@
 	"level": "#%%rank%% | Level %%level%% | XP: %%xp%%",
 	"youIndicator": "YOU",
 	"thousand": "k",
+	"million": "m",
```

That's not how we do millions
```suggestion
	"million": "M",
```